src/utils/types.ts

```diff
@@ -1,4 +1,5 @@
 export type CreateUserType = {
   username: string;
   email: string;
+  role: string;
```

Comment:
You can have this as an enum of specified roles tho, so that it won't receive any string